features/step_definitions/sign_in_steps.rb

```diff
@@ -0,0 +1,9 @@
+Given(/^I am on the sign in page$/) do
+  visit new_user_session_path
+end
+
+Given(/^the following user exist$/) do |table|
+  table.hashes.each do |hash|
+    User.create!(name: hash[:name] , email: hash[:email] , password: hash[:password])
```

Comment:
What happened with using factories? A bit of a newbie mistake guys! 😵

Also if you skip the password_confirmation in your table you can use 
```
create(:user, hash)
```
Short and neat.